.travis.yml

```diff
@@ -1,49 +0,0 @@
-language: php
-
-sudo: false
-
-# Test main supported versions of PHP against latest WP. 5.2 is min supported PHP version.
-php:
-  - 5.6
```

Comment:
@diegocurbelo - I noticed we're not testing with PHP 5.6 anymore. Was it intentional to drop tests for 5.6?

Reply:
That was moved to the compatibility GitAction Workflow, which I did not include in this PR.

I'll add it, with the following oldest dependencies check:
```
- woocommerce: '3.3.0'
- wordpress:   '4.7'
- php:         '5.6'
```